xbmc/music/MusicDatabase.cpp

```diff
@@ -3689,6 +3698,8 @@ bool CMusicDatabase::GetSongsNav(const std::string& strBaseDir, CFileItemList& i
 
 void CMusicDatabase::UpdateTables(int version)
 {
+  CLog::Log(LOGINFO, "%s - updating tables", __FUNCTION__);
```

Comment:
This is kinda dupe (`CDatabase::UpdateVersion` already logs) and most likely a left over from debugging.